app/src/main/java/br/com/dionataferraz/vendas/data/entity/User.kt

```diff
@@ -0,0 +1,13 @@
+package br.com.dionataferraz.vendas.data.entity
+
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+
+@Entity(tableName = "user")
+data class User(
```

Comment:
é bom sempre separa as classes por sufixos, nesse caso estamos representando um banco de dados, então devemos chamar de Entity
```suggestion
data class UserEntity(
```